### PR TITLE
docs: fix articles archive/unarchive an environment (#884) 

### LINKED ISSUE #884: [docs]: fix articles "Archive/Unarchive an Environment"
### Description

Article [Archive an Environment](https://www.devopness.com/docs/environments/archive-environment/) and [Unarchive an Environment](https://www.devopness.com/docs/environments/unarchive-environment/) needs to be updated

#### Archive button moved to Settings

Now, when a user wants to archive/unarchive an environment, they need to click the Settings button and select the option from the dropdown menu, as highlighted in the picture below

image

### Issue List

- [ ] Update "Select a project" step
```diff
-  1. Select a Project
+  1. In the list of Projects find the Project with the Team you want to manage and click View
```
- [ ] Remove "Find the Environments card" step, removing duplicated step
```diff
-  1. Find the Environments card
   1. Click View in the Environments card to see a list of existing Environments
```
- [ ] Guide the user to click SETTINGS
```diff
-  1. On the upper-right corner of the Environment resources list, click ...
+  1. On the upper-right corner of the Environment resources list, click SETTINGS
```

### PR DESCRIPTION
**Pull request by oliverigor27**
## Description of changes

This commit addresses the "Archive/Unarchive an Environment" issue by updating the documentation.

- [x]  Update "Select a project" step
- [x]  Remove "Find the Environments card" step, removing duplicated step
- [x]  Guide the user to click SETTINGS

## GitHub issues resolved by this PR

- [ ] closes #884

## Quality Assurance

- Once the changes in this PR are merged and deployed, success criteria is: 
- Add clear instructions for the users who wants to archiving/unarchiving environments.

### CODE REVIEW

**Review by thlmenezes — CHANGES_REQUESTED** [review]
Thank you for contributing @oliverigor27

Left some suggestions below to improve the article with your help

File: docs/environments/unarchive-environment.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ links:
 
 1. On Devopness, access the `Environment` resources list of the archived `Environment` you want to unarchive
     > Follow the {% mentionPost "/docs/environments/find-archived-environment" %} guide
-1. On the upper-right corner of the `Environment` resources list, click `...`
+1. On the upper-right corner of the `Environment` resources list, click **SETTINGS**
```
[the remark this answers is not in the document]
Comment (oliverigor27) [reply docs/environments/unarchive-environment.md:16]: Ok, i'm gonna do that

File: docs/environments/archive-environment.md [hunk 2]
```diff
@@ -26,10 +26,9 @@ The project owner can unarchive an archived environment, making it active and ac
...
 
 1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
-1. Select a `Project`
-1. Find the `Environments` card
+1. In the list of `Projects` find the Project with the Team you want to manage and click View
 1. Click `View` in the `Environments` card to see a list of existing `Environments`
 1. Find the `Environment` you want to archive and click `View`
-1. On the upper-right corner of the `Environment` resources list, click `...`
+1. On the upper-right corner of the `Environment` resources list, click **SETTINGS**
 1. Use the drop-down menu to choose `Archive`
```
[the remark this answers is not in the document]
Comment (oliverigor27) [reply docs/environments/archive-environment.md]: Done!

File: docs/environments/unarchive-environment.md [hunk 3]
```diff
@@ -13,6 +13,6 @@ links:
...
 1. Use the drop-down menu to choose `Unarchive`
```
[the remark this answers is not in the document]
Comment (oliverigor27) [reply docs/environments/unarchive-environment.md]: Done!

File: docs/environments/archive-environment.md [hunk 4]
```diff
@@ -26,10 +26,9 @@ The project owner can unarchive an archived environment, making it active and ac
 {% endnote %}
 
 1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
-1. Select a `Project`
-1. Find the `Environments` card
+1. In the list of `Projects` find the Project with the Team you want to manage and click View
 1. Click `View` in the `Environments` card to see a list of existing `Environments`
 1. Find the `Environment` you want to archive and click `View`
-1. On the upper-right corner of the `Environment` resources list, click `...`
+1. On the upper-right corner of the `Environment` resources list, click **SETTINGS**
```
[the remark this answers is not in the document]
Comment (oliverigor27) [reply docs/environments/archive-environment.md]: Done!

**Review by thlmenezes — APPROVED** [review]

**Comment by jfoliveira** [comment]
@cla-bot check

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**